Cargo.toml

```diff
@@ -28,6 +28,9 @@ default-members = [
 # https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html
 resolver = "2"
 
+[patch.crates-io]
+op-alloy-consensus = { git = "https://github.com/alloy-rs/op-alloy", branch = "rf/as-ref-tx-env" }
+
```

Comment:
This needs to be fixed once a release is cut with https://github.com/alloy-rs/op-alloy/pull/464

Reply:
The `cargo-deny` ci job will fail until this is removed